v4/export/sql.go

```diff
@@ -77,6 +77,40 @@ func ListAllTables(db *sql.DB, database string) ([]string, error) {
 	return tables.data, nil
```

Comment:
Should we delete `ListAllTables` and `ListAllViews`?

Reply:
we could if nothing uses them. doesn't really matter.